src/components/Book.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import './Book.css';
```

Comment:
- [ ] Please, It might be a good idea to not disable linter rules. We use them to avoid errors, bugs, and other issues in our code. ♻️

Reply:
Hello, thank you for your comments, 
With enabled linter I get this error: 
![image](https://user-images.githubusercontent.com/77038610/144960265-ced00899-deec-4b83-97c3-eada10bcfba7.png)
even though book props is in props validation : 
![image](https://user-images.githubusercontent.com/77038610/144960350-6c4866e6-a541-4c5a-bb16-c73befbb6c2c.png)

Please tell me how I could resolve this issue.

Reply:
- Please, It would be better to look for information about prop types like [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html#gatsby-focus-wrapper)
- Prop types are a validation for the props in your components. There are some validations to check if your prop is a number, string, or another type.
- Note: Don't forget to install the dependency for `prop-types`. 😉

Reply:
Thank you, after defining propTypes linter didn't show error 👍🏻